src/main/java/forms/Form.java

```diff
@@ -1,4 +1,23 @@
 package forms;
 
-public class Form {
+public abstract class Form {
+    private String title;
+    private String submit;
```

Comment:
what is this attribute used for? i feel like it should be a reference to whatever function we want to call after the form is submitted, like a callback, but probs not a string, unless there's some use for it i'm not aware of